ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -256,9 +256,12 @@ def create_optional_operators_catalogsource_non_ga(force=False):
     if config.DEPLOYMENT.get("optional_operators_image"):
         for _dict in optional_operators_data:
             if _dict.get("kind").lower() == "catalogsource":
-                _dict["spec"]["image"] = config.DEPLOYMENT.get(
-                    "optional_operators_image"
-                )
+                if not config.DEPLOYMENT.get("optional_operators_image"):
```

Comment:
This "negative" condition will never be met - since the whole block is in `if config.DEPLOYMENT.get("optional_operators_image"):` condition.

Reply:
Have addressed this in latest commit